packages/config/src/config/moonbeam/assets/glmr.ts

```diff
@@ -55,7 +55,7 @@ export const GLMR: MoonbeamXcmConfig = {
         .limitedReserveWithdrawAssets()
         .successEvent(PolkadotXcmExtrinsicSuccessEvent.Attempted)
         .origin(astar)
-        .V1()
+        .V1V2()
```

Comment:
I thought if we are not passing any params to `.V1V2()` and there is no other options maybe remove this step at all?

Reply:
We can remove it, but maybe it's better to keep it so it's consistent with the extrinsics. I don't have strong opinions

Reply:
Also, `.V1V2X2()` definitely looks weird